ouroboros-consensus/test/infra-test/Test/Ouroboros/Consensus/ChainGenerator/Tests/Adversarial.hs

```diff
@@ -42,6 +42,8 @@ import qualified Test.Tasty.QuickCheck as TT
 
 tests :: [TT.TestTree]
 tests = [
+    TT.testProperty "k+1 blocks after the intersection" prop_kPlus1BlocksAfterIntersection
+  ,
     TT.testProperty "prop_adversarialChain" prop_adversarialChain
   ,
     TT.localOption (TT.QuickCheckMaxSize 14) $ TT.testProperty "prop_adversarialChainMutation" prop_adversarialChainMutation
```

Comment:
While we're at it, do you happen to know what `prop_adversarialChain` and `prop_adversarialChainMutation` are supposed to test? Could we find a more description test name here? (Only if you can do that from the top of your head!)

Reply:
`prop_adversarialChain` checks that the density is lower than in the honest chain, and also checks Length of competing chains.

`prop_adversarialChainMutation` checks that when decreasing the security parameter or when increasing the message delay, an alternative chain generated with the original values might fail the checks in `prop_adversarialChain`.

I'll try to rename.

Reply:
Renamed in 617fff17